skyvern/forge/sdk/api/llm/api_handler_factory.py

```
@@ -270,8 +271,62 @@ async def llm_api_handler_with_router_and_fallback(
+                    if "vertex_ai/" in llm_config.model_name:
+                        caching_system_message = {
+                            "role": "system",
+                            "content": [
+                                {
+                                    "type": "text",
+                                    "text": context_cached_static_prompt,
+                                    "cache_control": {"type": "ephemeral", "ttl": "3600s"},
+                                }
+                            ],
+                        }
+                        messages = [caching_system_message] + messages
```

Ruff (0.13.1)

Consider `[caching_system_message, *messages]` instead of concatenation

Replace with `[caching_system_message, *messages]`

(RUF005)